draft-ietf-anima-brski-prm.md

```diff
@@ -1103,589 +1171,599 @@ pledgeenrollrequesttrigger = {
 
 $enroll-type /= "enroll-generic-cert"
 ~~~~
-{: #tper_CDDL_def title='CDDL for Pledge Enroll-Request Trigger' artwork-align="left" sourcecode-markers=“true”}
+{: #tper_CDDL_def title='CDDL for Pledge Enroll-Request Trigger (pledgeenrollrequesttrigger)' artwork-align="left"}
 
-The enroll-type  allows for specifying arbitrary indications, which type of certificate is to be enrolled. 
-BRSKI enris an enum, identifying what is being enrolled. 
-As shown in {{tper_CDDL_def}}, BRSKI-PRM defines only "enroll-generic-cert" for the enrollment of the generic LDevID certificate. 
-Other specifications using this mechanism may define further values, e.g., to bootstrap application related certificates, e.g., indicated by a value "enroll-app-cert".
+The `enroll-type` member allows for specifying which type of certificate is to be enrolled.
+As shown in {{tper_CDDL_def}}, BRSKI-PRM only defines the enum value `enroll-generic-cert` for the enrollment of the generic, device-related LDevID certificate.
+Other specifications using this artifact may define further enum values, e.g., to bootstrap application-related EE certificates with addtional CSR attributes.
 
-The Pledge Enroll-Request Trigger (tPER) artifact MUST be encoded in JSON as defined in {{!RFC8259}} following the CDDL definition {{tper_CDDL_def}}.
+TODO(Create an IANA registry for the enroll-type enum?)
```

Comment:
Should we include this in the IANA Considerations?
It would make managing extensions much easier.

Reply:
That is likely a good idea. As we only provide one for the generic cert but leave it open for other to define other types. This may become important when the intention is to trigger also for re-enrollment or for application specific certificates.

Reply:
New IANA text in chat, including in IANA Considerations.